doc/source/parameters/dem/model_parameters.rst

```diff
@@ -0,0 +1,92 @@
+Model Parameters
+-------------------
+In this subsection, DEM simulation parameters are defined. These parameters include ``contact detection method`` and its subsequent information (``dynamic contact search size coefficient`` **or** ``contact detection frequency`` for ``dynamic`` **or** ``constant`` contact detection method), ``repartition method`` (``once``, ``frequent`` or ``dynamic``) for ``load balancing`` (in parallel simulations), ``particle weight`` which defines the particle weight compared to a default ``cell weight`` of 1000, ``neighborhood threshold`` (which defines the contact neighbor list size: ``neighborhood threshold`` * particle diameter), ``particle particle contact force method``, ``particle wall contact force method`` and ``integration method``. 
```

Comment:
I would explain these model parameters after the sections is introduced. Or otherwise, you can explain it in sentences outside of notes. The goal is to be as didactic as possible.

Reply:
moved.